Rules/CommonScripts/PropagateBans.as

```diff
@@ -0,0 +1,12 @@
+// This script works with an external program that propagates bans to other connected servers
+// This ensures banned players get banned on all other connected servers
+
+void onBan(const string username, const int minutes, const string reason)
+{
+	tcpr("BAN " + username + " " + minutes + " " + reason);
```

Comment:
calling tcpr client side on mac or linux results in a crash, stick the script in #define server_only or remove script client side onInit